packages/db/src/translate/providers.ts

```diff
@@ -11,6 +12,29 @@ export const encryptValueWithKMS = (
   return encryptFunction(providerKmsKeyId, value);
 };
 
+export const translatePostgresProviderToApiProvider = (
+  record: PostgresProviderRecord
+): ApiProvider => {
+  const apiProvider = {
+    id: record.name,
+    cmKeyId: record.cm_key_id,
+    certificateUri: record.certificate_uri,
+    privateKey: record?.private_key,
```

Comment:
😯 I didn't know about this syntax. Neat!

Reply:
👍🏻    https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining 

In this instance, though, this is a mistake as we know `record` will always be an object of type `PostgresProviderRecord` for any reasonable use of this method - as such, the .? is redundant.    8d9a17956